apps/challenges/views.py

```diff
@@ -959,12 +944,9 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
                 'error': 'Error in creating challenge. Please check the yaml configuration!'}
             return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
         finally:
-            try:
-                shutil.rmtree(BASE_LOCATION)
-                logger.info('Zip folder is removed')
-            except:
-                logger.exception('Zip folder for challenge {} is not removed from location'.format(challenge.pk,
-                                                                                                   BASE_LOCATION))
+            shutil.rmtree(BASE_LOCATION)
```

Comment:
We should catch this exception. If this operation fails, then we should throw an error, no?

Reply:
Sure :)